 When gnome-spinner-rest theme icon can't be found don't complain
 and use the main galeon window icon which is slightly less inappropriate
 than an empty lookin space on the toolbar.

---
 src/galeon-spinner.c |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/galeon-spinner.c b/src/galeon-spinner.c
index 519caad..860641a 100644
--- a/src/galeon-spinner.c
+++ b/src/galeon-spinner.c
@@ -199,7 +199,8 @@ select_spinner_image (GaleonSpinner *spinner)
 	GList *element;
 
 	if (spinner->details->timer_task == 0) {
-		return g_object_ref (spinner->details->quiescent_pixbuf);
+		return (spinner->details->quiescent_pixbuf != NULL) ?
+		        g_object_ref (spinner->details->quiescent_pixbuf) : NULL;
 	}
 	
 	if (spinner->details->image_list == NULL) {
@@ -594,8 +595,7 @@ galeon_spinner_load_images (GaleonSpinner *spinner)
 					   wanted_size, 0, NULL);
 
 	if (pixbuf == NULL) {
-		g_warning ("Spinner rest icon not found");
-		return;
+        	pixbuf = gdk_pixbuf_new_from_file (DATADIR "/pixmaps/galeon.png", NULL);
 	}
 
 	spinner->details->quiescent_pixbuf = 
-- 
1.6.5.2

